.github/workflows/verify.yaml

```diff
@@ -63,6 +63,28 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
 
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache yarn cache
+        uses: actions/cache@v2
+        id: cache-yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Cache node_modules
```

Comment:
https://github.com/actions/cache/blob/main/examples.md#node---npm

Apparently it's not recommended to cache the `node_modules` directory itself. I guess as we're using Yarn, we should just use that cache source for now.

Reply:
You should just be able to remove the node bits and leave the yarn bits, then the PR will be fine.

Thanks

Reply:
I removed the caching of the node_modules :)